provider/aws/aws.go

```diff
@@ -679,11 +679,15 @@ func (p *AWSProvider) AdjustEndpoints(endpoints []*endpoint.Endpoint) []*endpoin
 			}
 		} else {
 			alias = useAlias(ep, p.preferCNAME)
-			log.Debugf("Modifying endpoint: %v, setting %s=%v", ep, providerSpecificAlias, alias)
+			log.Debugf("Modifying endpoint: %v, unsetting TTL", ep)
```

Comment:
This logging change appears to be inaccurate.

Reply:
You may have missed the refresh, unsetting should be fine?

Reply:
That is very incorrect, fixing now!